src/styles/blocks/card.scss

```diff
@@ -0,0 +1,116 @@
+.card {
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  align-items: center;
+  font-size: 10px;
+  min-width: 1000px;
+  width: getWidth(4, $cards-width, $cards-vertical_shift);
+  margin: 0 auto;
+
+  &__container {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border: 1px solid #f3f3f3;
+    border-radius: 5px;
+    width: $cards-width;
+    padding: 32px 16px 16px;
+    margin-left: 48px;
+    margin-top: 46px;
+
+    &--margin {
+      margin-top: 50px;
+    }
+
+    &:first-child {
+      margin-left: auto;
+    }
+
+    &:nth-child(5n) {
+      margin-left: auto;
+    }
+
+    &:nth-child(4n) {
+      margin-right: auto;
+    }
+  }
+
+  &__img {
+    margin: 0 auto;
+    align-self: center;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__name {
+    margin-top: 40px;
+    bottom: 165px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main_text-color;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400px;
```

Comment:
it's invalid . Check other cases
<img width="172" alt="Screenshot 2023-04-06 at 20 48 55" src="https://user-images.githubusercontent.com/117587463/230456660-268a89e4-1a2c-4aff-b20b-fd0ef602b656.png">
```suggestion
    font-weight: 400;
```